config/assignment.go

```diff
@@ -277,9 +277,9 @@ func seeder(assignmentKey string) *Seeder {
 		return nil
 	}
 
-	toBranch, ok := seederMap["toBranch"]
-	if !ok {
-		toBranch = "master"
+	toBranch := "master"
```

Comment:
use `main` please, since LRZ GitLab has recently changed from `master` to `main`

Reply:
Already changed in the last commit 😬